docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -554,7 +554,7 @@ fields:
     choices: fields
     object labeler: |
       lambda y: bold(y.variable) + "[BR]" + y.label
-  - Add a "continue button field" on this screen: questions[i].has_mandatory_field
+  - Override the default logic flow by adding a "continue button field": questions[i].has_mandatory_field
     datatype: noyes
     hide if: questions[i].is_informational_screen
     help: |
```

Comment:
I'd also suggest adding more of the [explanation that Quinten gave](https://github.com/SuffolkLITLab/docassemble-assemblylinewizard/issues/128#issuecomment-783508680) to the help info. Since the help bubble is just a little overlay on the screen and not a brand new screen, maybe just taking the last sentence would be good. It's a help screen, so I feel that more information is better than less. 

> It helps you give a more semantically meaningful item in the interview order block when the variables listed on the screen don't match the purpose of displaying the screen. It adds a custom variable to the interview order block rather than the first field on the screen.
>
>You might do this if you want to go back in and add show/hide logic on that screen, which could otherwise break the interview flow.

Reply:
Thanks! Added!